.golangci.yaml

```diff
@@ -14,6 +14,9 @@ run:
 
   # include test files or not, default is true
   tests: true
+  # list of build tags, all linters use it. Default is empty list.
+  build-tags:
```

Comment:
Nice, I didn't realize I could simply include the build tags here.

Reply:
Thanks :)